src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java

```diff
@@ -145,6 +155,33 @@ public TableDescription deleteTable(String tableName) {
         return tableDescription;
     }
 
+    @Override
+    public Map<TenantTable, CreateTableRequest> getAllMtTables() {
+        ScanRequest scanReq = new ScanRequest(tableDescriptionTableName);
+        Map<TenantTable, CreateTableRequest> ret = Maps.newHashMap();
+        ScanResult scanResult;
+        scanResult = amazonDynamoDb.scan(scanReq);
+        ret.putAll(scanResult.getItems().stream()
+            .collect(Collectors.toMap(
+                rowItem ->
+                    getTenantTableFromHashKey(rowItem.get(tableDescriptionTableHashKeyField).getS())
+                ,
+                rowItem -> {
+                    String tableDataJson = rowItem.get(tableDescriptionTableDataField).getS();
+                    return getCreateTableRequest(jsonToTableData(tableDataJson));
+                })));
+
+        if (scanResult.getCount() == scanReq.getLimit()) {
```

Comment:
will need to get to this obviously, but this will work until I try backing up a table with more tables than are returned in a single scan request